### PR TITLE
Feature/str68

### PR DESCRIPTION
**Pull request by jamespilcher**
Design + CSS changes

Made it easier to change fonts (mostly moving CSS changes to base.html)
Imported custom fonts (see base.html)
Did (temporary) redesign of leaderboard and landing page
Reworded a few things here and there.

### CODE REVIEW

File: streatham_go/app/templates/app/index.html
Comment (daniel-moulton) [inline streatham_go/app/templates/app/index.html:1]: Can you make all the carousel images the same size please!

**Review by daniel-moulton — CHANGES_REQUESTED** [review]
Just need to change the image carousel, but good work

File: streatham_go/app/templates/app/index.html
Comment (jamespilcher) [reply streatham_go/app/templates/app/index.html:1]: They are placeholder!!!!!. 

The way carousel images work make it so its stupidly difficult to have them all crop to the same size (AFAIK!!). They are all set to the same width though, as long as each image has the same aspect ratio it'll be fine,

**Review by AlexGulliver — APPROVED** [review]
lgtm

**Review by daniel-moulton — APPROVED** [review]
lgtm

**Pull request merged**